code/index.html

```diff
@@ -1,30 +1,113 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
```

Comment:
Two body tags? And the accordion is placed above the second head :-) Needs some structure!
Maybe a good thing would be to remove the Technigo comments and add your own.